### PR TITLE
[PIR]Migrate Sparse API softmax N0.3

### PR DESCRIPTION
**Pull request by epiphanyer**
### PR Category

Execute Infrastructure

### PR Types

Others

### Description

为paddle.nn.functional.softmax算子添加PIR模式下静态图单测。
issue: #64492
pcard-67164

### CODE REVIEW

**Review by risemeup1 — APPROVED** [review]
LGTM
